src/components/BuildDetails.tsx

```diff
@@ -244,6 +252,24 @@ class BuildDetails extends React.Component<Props> {
     });
   }
 
+  getAndCancelAll(build) {
+    for (let i = 0; i < build.tasks.length; i++) {
+      let task = build.tasks[i];
+      const variables = {
+        input: {
+          clientMutationId: 'abort-' + task.id,
+          taskId: task.id,
+        },
+      };
+
+      commitMutation(environment, {
```

Comment:
This functionality should be in a separate mutation supported by API and not `O(n)` calls to the API

Reply:
I changed it to use abort, just need help fixing relay now

Reply:
Update: I removed it for now so this can be shipped.